docs/snippets/common/storybook-babel-configuration-example.ts.mdx

```diff
@@ -1,12 +1,14 @@
 ```ts
-// mithril/src/server/framework-preset-mithril.ts
+// app/mithril/src/server/framework-preset-mithril.ts
+
+import { TransformOptions } from '@babel/core';
 
 export function babelDefault(config: TransformOptions) {
   return {
     ...config,
     plugins: [
-      ...config.plugins, 
-      require.resolve('@babel/plugin-transform-react-jsx')
+      ...config.plugins,
+      [require.resolve('@babel/plugin-transform-react-jsx'), {}, 'preset'],
```

Comment:
Is this change related?

Reply:
I was checking the changes in the repo for mithril and that caught my eye. Compared it with what's currently documented and updated it accordingly so that we're on the safe side. I can always revert it back